From af85c9706be507f53150a2c27478e4e35285871b Mon Sep 17 00:00:00 2001
From: Self-Hosting-Group
 <155233284+Self-Hosting-Group@users.noreply.github.com>
Date: Fri, 14 Jun 2024 08:44:29 +0200
Subject: [PATCH] Change default upnp_ssdp_interval to 900s

to follow specification and upstream change
https://github.com/miniupnp/miniupnp/commit/9339f0e52c7a1d2024377e7a2446e80b93d334aa

(cherry picked from commit f0a1bd8b32b8269ef32021fa83f308b5417d1615)
---
 release/src/router/shared/defaults.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src/router/shared/defaults.c b/release/src/router/shared/defaults.c
index 05bca2c1cd..98e416cde0 100644
--- a/release/src/router/shared/defaults.c
+++ b/release/src/router/shared/defaults.c
@@ -2635,7 +2635,7 @@ struct nvram_tuple router_defaults[] = {
 #endif
 	{ "upnp_secure", "1", CKN_STR1, CKN_TYPE_DEFAULT, CKN_ACC_LEVEL_DEFAULT, CKN_ENC_DEFAULT, 0 },
 	{ "upnp_port", "0", CKN_STR5, CKN_TYPE_DEFAULT, CKN_ACC_LEVEL_DEFAULT, CKN_ENC_DEFAULT, 0 },
-	{ "upnp_ssdp_interval", "60", CKN_STR5, CKN_TYPE_DEFAULT, CKN_ACC_LEVEL_DEFAULT, CKN_ENC_DEFAULT, 0 },
+	{ "upnp_ssdp_interval", "900", CKN_STR5, CKN_TYPE_DEFAULT, CKN_ACC_LEVEL_DEFAULT, CKN_ENC_DEFAULT, 0 },
 	{ "upnp_mnp", "1", CKN_STR1, CKN_TYPE_DEFAULT, CKN_ACC_LEVEL_DEFAULT, CKN_ENC_DEFAULT, 0 },
 	{ "upnp_clean", "1", CKN_STR1, CKN_TYPE_DEFAULT, CKN_ACC_LEVEL_DEFAULT, CKN_ENC_DEFAULT, 0 },
 	{ "upnp_clean_interval", "600", CKN_STR5, CKN_TYPE_DEFAULT, CKN_ACC_LEVEL_DEFAULT, CKN_ENC_DEFAULT, 0 },
-- 
2.25.1

